.pipelines/v2/templates/job-build-project.yml

```diff
@@ -228,88 +229,31 @@ jobs:
           batchSignPolicyFile: '$(build.sourcesdirectory)\.pipelines\ESRPSigning_abstracted_utils_dll.json'
           ciPolicyFile: '$(build.sourcesdirectory)\.pipelines\CIPolicy.xml'
 
-  - task: VSBuild@1
```

Comment:
this is where dart CI complains about preview packages. so I removed these. The plan is to remove these from upstream as well, some time soon, so not a big deal. It can go in here IMHO